virtcontainers/pkg/oci/utils.go

```diff
@@ -775,6 +775,14 @@ func addAgentConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig) error
 		c.TraceType = value
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.AgentContainerPipeSize]; ok {
+		containerPipeSize, err := strconv.ParseUint(value, 10, 32)
+		if err != nil {
+			return fmt.Errorf("Error parsing annotation for agent.container_pipe_size: Please specify uint32 value")
```

Comment:
This error can make use of the `const` exported by the annotations package to avoid hard-coding the string.

Reply:
Done!